test/test-v8debugapi.js

```diff
@@ -587,8 +601,13 @@ describe('v8debugapi', function() {
 
           var topFrame = bp.stackFrames[0];
           assert.equal(topFrame['function'], 'foo');
-          assert.equal(topFrame.arguments[0].name, 'n');
-          assert.equal(topFrame.arguments[0].value, '3');
+          if (semver.satisfies(process.version, '<1.6')) {
```

Comment:
/cc @ofrobots thoughts on this dichotomy between versions? If the debug team has agreed that arguments and locals are handled uniformly by the UI this should be fine.